kotlin-mobile-jetpackCompose-new/app/src/main/java/com/example/kotlin_android_jetpack_conmpose_new/MainActivity.kt

```diff
@@ -0,0 +1,113 @@
+                            odświeżania się ekranu aplikacji
+                            Zmienna powinna na początku być pustym Stringiem
+
+                            Podpowiedź:
+                            Trochę o stanie plus 3 sposoby na deklarowanie zmiennych
+                            https://developer.android.com/jetpack/compose/state
+                        */
+                        Text(text = "Rekrutacja")
+                        Spacer(modifier = Modifier.height(4.dp))
+                        TextField(
+                            value = "",
+                            onValueChange = { //todo 1.b przypisz tu swojej zmiennej wartość
+                                // it czyli lokalnej zmiennej tekstowej komponentu
+                                            //przypisz swoją zmienną do value
+                                },
+                            maxLines = 1
+                        )
+                        Spacer(modifier = Modifier.height(4.dp))
+                        val context = LocalContext.current
+                        Button(onClick = { } ) {
+                            Text(text = "Przycisk")
+                        }
+                    }
+                    /*2.TODO Drugim z zadań jest implementacja metody, w taki sposób, aby
+                          kliknięcie przycisku powodowało otwarcie activity SecondActivity, gdy
+                          wpisanym przez użytkownika tekstem jest 'AKAI'.
+                          Kolejne zadanie znajdziesz poniżej.
+
+                          Podpowiedź poczytaj o Inteny
+                   */
+//
+//                    3.a TODO Kolejnym zadaniem jest wstawienie nowego przycisku
+//                        do tego widoku. Przycisk należy wstawić tak, aby
+//                    wyświetlał się pomiędzy EditTextem, a przyciskiem o z napisem "Przycisk",
+//                    Dodatkowo na przycisku ma wyświetlać się napis 'Kliknij mnie'.
+//                    Kolejny podpunkt znajdziesz w poniżej :)
+//                    Aby zachować spójność i estetykę widoku dla użytkowników możesz dodać również
+//                    odstęp mi®dzy przyciskiem a innymi elementami
+//                    możesz użyć gotowych componentów toolkitu Jetpack Compose
+//                    lub parametrów obiektu modifier
+//                    https://developer.android.com/jetpack/compose/modifiers
+
+
+                }
+            }
+        }
+    }
+
+    /*3.b TODO Druga część tego zadania polega na implementacji metody, która zmienia
+            wszyskie litery we wpisanym przez użytkownika tekscie na wielkie i
+            wstawienie przerobionego tekstu do TextFielda. Nie zapomnij dodać tej moetody
+            do utworzonego przez Ciebie przycisku
+            Kolejne zadanie czeka na Ciebie w pliku SecondActivity
+     */
+    private fun onYourButtonClick() {
+
+    }
+}
+
+@Composable
+fun Greeting(name: String) {
```

Comment:
Chcemy to zostawiac jesli jest to uzywane tylko w preview? Moze lepiej do preview dac aktualny ekran?